static/js/publisher/release/components/releasesTableCell.js

```diff
@@ -31,13 +31,13 @@ class ReleasesTableCell extends Component {
   renderRevision(revision, isPending) {
     return (
       <Fragment>
-        {isInDevmode(revision) && (
-          <span className="p-release-data__icon">
-            <DevmodeIcon revision={revision} showTooltip={false} />
-          </span>
-        )}
         <span className="p-release-data__info">
           <span className="p-release-data__title">{revision.revision}</span>
+          {isInDevmode(revision) && (
+            <span className="p-release-data__icon u-float-right">
```

Comment:
The `p-release-data__icon` class doesn't seem to be needed anymore (removed from styles).